code/modules/mining/machine_vending.dm

```diff
@@ -5,57 +5,58 @@
+	new /datum/data/mining_equipment("Item-Warp Pack",				/obj/item/extraction_pack,									25,					600),
 	new /datum/data/mining_equipment("Drone Health Upgrade", 		/obj/item/device/mine_bot_upgrade/health,					20,					600),
-	new /datum/data/mining_equipment("RFD M-Class",             	/obj/item/rfd/mining,								10,					600),
-	new /datum/data/mining_equipment("Brute First-Aid Kit",			/obj/item/storage/firstaid/brute,					30,					600),
-	new /datum/data/mining_equipment("Ore Magnet",					/obj/item/oremagnet,									10,					600),
+	new /datum/data/mining_equipment("RFD M-Class",             	/obj/item/rfd/mining,										10,					600),
+	new /datum/data/mining_equipment("Brute First-Aid Kit",			/obj/item/storage/firstaid/brute,							30,					600),
+	new /datum/data/mining_equipment("Ore Magnet",					/obj/item/oremagnet,										10,					600),
 	new /datum/data/mining_equipment("Minecart",					/obj/vehicle/train/cargo/trolley/mining,					-1,					600,	1),
-	new /datum/data/mining_equipment("Resonator",					/obj/item/resonator,									10,					700),
+	new /datum/data/mining_equipment("Resonator",					/obj/item/resonator,										10,					700),
 	new /datum/data/mining_equipment("Jaunter",						/obj/item/device/wormhole_jaunter,							20,					750),
-	new /datum/data/mining_equipment("Mining RIG",					/obj/item/rig/industrial,							5,					1000),
-	new /datum/data/mining_equipment("Mass Driver",					/obj/item/mass_driver_diy,							5,					800),
+	new /datum/data/mining_equipment("Mining RIG",					/obj/item/rig/industrial,									5,					1000),
+	new /datum/data/mining_equipment("Mass Driver",					/obj/item/mass_driver_diy,									5,					800),
 	new /datum/data/mining_equipment("Mining Drone",				/mob/living/silicon/robot/drone/mining,						15,					800),
 	new /datum/data/mining_equipment("Minecart Engine",				/obj/vehicle/train/cargo/engine/mining,						-1,					800,	1),
 	new /datum/data/mining_equipment("Drone KA Upgrade",			/obj/item/device/mine_bot_upgrade/ka,						10,					800),
-	new /datum/data/mining_equipment("Ore Summoner",				/obj/item/oreportal,									35,					800),
-	new /datum/data/mining_equipment("Lazarus Injector",			/obj/item/lazarus_injector,							25,					1000),
+	new /datum/data/mining_equipment("Ore Summoner",				/obj/item/oreportal,										35,					800),
+	new /datum/data/mining_equipment("Lazarus Injector",			/obj/item/lazarus_injector,									25,					1000),
 	new /datum/data/mining_equipment("Industrial Drill Head",		/obj/machinery/mining/drill,								-1,					1000,	1),
-	new /datum/data/mining_equipment("Super Resonator",				/obj/item/resonator/upgraded,						10,					1250),
-	new /datum/data/mining_equipment("Diamond Pickaxe",				/obj/item/pickaxe/diamond,							10,					1500),
-	new /datum/data/mining_equipment("Thermal Drill",				/obj/item/gun/energy/vaurca/thermaldrill,			5,					3750)
+	new /datum/data/mining_equipment("Orbital Drill Dropper",		/obj/item/device/orbital_dropper/drill,						10,					1250),
```

Comment:
Bump up the price of that to 3000+.
Drillhead + 2 braces is 2000 (which would be more than the price of a complete drill with orbital drop)